pkgs/misc/vim-plugins/default.nix

```diff
@@ -439,6 +439,19 @@ rec {
     };
   };
 
+  vim-jinja = buildVimPlugin {
+    name = "vim-jinja";
+    src = fetchgit {
```

Comment:
please use fetchFromGitHub instead to fetch an archive for the rev from over the GitHub api